index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main">
+      <div id="title">Dr. Norman Borlaug</div>
+
+      <div id="img-div">
+        <img id="image" src="image/norman-borlaug.jpg">
+        <figcaption id="img-caption">This is Norman Ernest Borlaug, known for the <a href="https://en.wikipedia.org/wiki/Green_Revolution">Green Revolution</a></figcaption>
+      </div>
```

Comment:
- [ ] You should use more semantic tags. Here, instead of using the `<div>` tag, you should have used the `<figure>` tag instead as it is more descriptive. 

Please look at the first example on this [MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)

Reply:
It would sound better if you put it this way: `It's always better to use semantic tags. Here, instead of using the <div> tag, you should have used the <figure> tag instead as it is more descriptive`. 
This doesn't sound like a command but a suggestion which is more polite.